routes/adminLogin.js

```diff
@@ -7,10 +7,11 @@ var route = {
 	get: function (req, res, next) {
 
         var linz = require('../'),
-        // check if resetPassword() is defined for user model
-        locals = {
-            hasResetPassword: linz.api.model.get(linz.get('user model')).sendPasswordResetEmail
-        };
+            loginError = req.flash('error'),
+            locals = {
+                hasResetPassword: linz.api.model.get(linz.get('user model')).sendPasswordResetEmail, // check if resetPassword() is defined for user model
+                loginErrorMsg: (!loginError.length) ? '' : loginError.toString() //set login error message
+            };
```

Comment:
Since `req.flash('error')` contains an array. You should simply pass this array to the template and update the template to handle an array of messages.